packages/server/initPublicPath.ts

```diff
@@ -0,0 +1,16 @@
+import appOrigin from './appOrigin'
+
+declare let __webpack_public_path__: string
```

Comment:
we can't set the public path at buildtime because different PPMIs will host their assets in different places.
since the public path is defined during predeploy, we must set it at runtime